server/src/main/java/org/candlepin/util/CrlFileUtil.java

```diff
@@ -133,18 +133,20 @@ public void initializeCRLFile(File file, Collection<BigInteger> revoke) throws I
      */
     public File stripCRLFile(File file) throws IOException {
         File tempFile = File.createTempFile("candlepin_crl_", ".pem");
-        BufferedReader r = new BufferedReader(new FileReader(file));
-        BufferedWriter w = new BufferedWriter(new FileWriter(tempFile));
 
+        BufferedReader r = null;
+        BufferedWriter w = null;
         String line = null;
 
         try {
+            r = new BufferedReader(new FileReader(file));
+            w = new BufferedWriter(new FileWriter(tempFile));
+
```

Comment:
Could use a multi-try-with-resources
```
try (r = new BufferedReader(new FileReader(file));
     w = new BufferedWriter(new FileWriter(tempFile))) {
```
to cut down the boilerplate.

Reply:
There was a reason I didn't at the time, but I can't think of that reason now. Guess it couldn't have been that important. Fixed.